### DISCUSSION TITLE
GITHUB Enterprise Trial - Setting up Azure AD SAML

### DISCUSSION THREAD

**Discussion by jarnold-a** [community/community#36722] [Enterprise] [answered]
I am following this article closely 
https://learn.microsoft.com/en-us/azure/active-directory/saas-apps/github-tutorial#create-github-test-user

But my SAML test is failing with the below error code.

Sign in
Sorry, but we’re having trouble signing you in.

AADSTS650056: Misconfigured application. This could be due to one of the following: the client has not listed any permissions for 'AAD Graph' in the requested permissions in the client's application registration. Or, the admin has not consented in the tenant. Or, check the application identifier in the request to ensure it matches the configured client application identifier. Or, check the certificate in the request to ensure it's valid. Please contact your admin to fix the configuration or consent on behalf of the tenant. Client app ID: dee2cdc2-7505-4746-9758-219fec5e3da0.

Anyone have any ideas?

**Comment by ossanna16** [comment]
Hey there @jarnold-a and thank you for asking a good question! I'm glad you found our community 🙂

**Comment by jarnold-a** [comment, marked as answer]
I found the issue. I had trailing slashes after the URLS on the Azure SAML config.
